Sofa/framework/Core/src/sofa/core/CollisionModel.h

```diff
@@ -25,8 +25,7 @@
 #include <sofa/core/objectmodel/BaseObject.h>
 #include <sofa/core/CollisionElement.h>
 
-//todo(dmarchal 2018-06-19) I really wonder why a collision model has a dependency to a RGBAColors.
-#include <sofa/type/RGBAColor.h>
+#include <sofa/type/RGBAColor_fwd.h>
```

Comment:
I'm surprise this is working. RGBAColor is used in a Data which as far as I remember ispreventing the use of forward declaration. Isn't RGBAColor included somewhere else ?

Reply:
Yep, you are right, too bad :(
I will revert that